dev-vcs/tortoisehg/tortoisehg-5.6.1.ebuild

```
@@ -0,0 +1,62 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+PYTHON_COMPAT=( python3_{6,7,8,9} )
```

```suggestion
PYTHON_COMPAT=( python3_{7..9} )
```
dev-python/iniparse doesn't support PYTHON_COMPAT python3_6

---

Thank you for advise. I thinked I need mask python3_6 as iniparse 0.4-r2 still has it. But as I know python3_6 is planning to be dropped soon, isn't it? Then it's really better to remove it here.

---

> I thinked I need mask python3_6 as iniparse 0.4-r2 still has it

There's no need to mask anything aside from limiting PYTHON_COMPAT as I believe portage will only attempt to install the latest version which fits within the constraints. If this package used python-single-r1 either version of `iniparse` would be okay (?) but because all python targets could be enabled, the version of iniparse selected needs to support every PYTHON_COMPAT here (in this case `dev-python/iniparse-0.5` is the only one that can). `dev-python/iniparse-0.42-r2` is esentially masked should `dev-vcs/tortoisehg` be installed. I believe I have that all correct

---

Done